### PR TITLE
fix: main README updates.

### PR DESCRIPTION
**Pull request by rjerrems**
Added changes from https://github.com/terraform-google-modules/terraform-example-foundation/pull/216
+ some additional fixes.

Branched from master to avoid clashes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-Cloud Build has been chosen to allow teams to quickly get started without needing to deploy a CI/CD tool, although it is worth noting the code can easily be executed by your preferred tool.
-Jenkins is also avaible to be used instead of Cloud Build.
+This is an example repo showing how the CFT Terraform modules can be composed to build a secure GCP foundation, following the [Google Cloud security foundations guide](https://services.google.com/fh/files/misc/google-cloud-security-foundations-guide.pdf).
+The supplied structure and code is intended to form a starting point for building your own foundation with pragmatic defaults you can customize to meet your own requirements. Currently, the step 0 is manually executed.
+From step 1 onwards, the Terraform code is deployed by leveraging either Google Cloud Build (by default) or Jenkins.
+Cloud Build has been chosen by default to allow teams to quickly get started without needing to deploy a CI/CD tool, although it is worth noting the code can easily be executed by your preferred tool.
```
Comment (daniel-cit) [inline README.md:5]: Which one is the recommend format `CI/CD tool` or `CICD tool` ?

File: README.md [hunk 2]
```diff
@@ -21,25 +22,30 @@ The bootstrap step includes:
...
-    ├── cft-cloudbuild
+    ├── cft-cloudbuild (prj-cicd if using Jenkins)
     └── cft-seed
 ```
 
-In addition, this step uses the optional Cloud Build submodule, which sets up Cloud Build and Cloud Source Repositories for each of the stages below.
-A simple trigger mechanism is configured, which runs a `terraform plan` for any non master branch and `terraform apply` when changes are merged to the master branch.
-Usage instructions are available in the bootstrap [README](./0-bootstrap/README.md).
+When this step uses the Cloud Build submodule, it sets up Cloud Build and Cloud Source Repositories for each of the stages below.
+Triggers are configured to run a `terraform plan` for any non environment branch branch and `terraform apply` when changes are merged to an environment branch (`development`, `non-production` & `production`).
```
Comment (daniel-cit) [inline README.md:47]: nit: `environment branch branch` =>  `environment branch`

File: README.md [hunk 3]
```diff
@@ -59,11 +65,12 @@ example-organization
 
 #### Logs
 
-Among the four projects created under the common folder, two projects (prj-c-logging, prj-c-org-billing-logs) are used for logging. The first one for organization wide audit logs and another for billing logs.
+Among the four projects created under the common folder, two projects (`prj-c-logging`, `prj-c-org-billing-logs`) are used for logging.
```
Comment (daniel-cit) [inline README.md:68]: They are six projects now:
- prj-c-billing-logs
- prj-c-dns-hub
- prj-c-interconnect
- prj-c-logging
- prj-c-scc
- prj-c-secrets


and `prj-c-org-billing-logs` is `prj-c-billing-logs`

File: README.md (hunk 1 above)
Comment (rjerrems) [reply README.md:5]: I will align all the references to CI/CD

**Comment by rjerrems** [comment]
Thanks for feedback @daniel-cit  - I have updated

File: README.md [hunk 4]
```diff
@@ -1,45 +1,51 @@
...
   - If using Cloud Build:
     - Cloud Source Repository
   - If using Jenkins:
+    - A GCE Instance configured as a Jenkins Agent
     - Custom Service Account to run Jenkins Agents GCE instances
     - VPN connection with on-prem (or where ever your Jenkins Master is located)
 
-It is a best practice to have two separate projects here (`cft-seed` and `prj-cicd`) for separation of concerns. On one hand, `cft-seed` stores terraform state and has the Service Account able to create / modify infrastructure. On the other hand, the deployment of that infrastructure is coordinated by a tool of your choice (either Cloud Build or Jenkins), which is implemented in `prj-cicd`.
+It is a best practice to separate concerns by having two projects here: one for the CFT resources and one for the CI/CD tool.
+The `cft-seed` project stores terraform state and has the Service Account able to create / modify infrastructure.
```
Comment (bharathkkb) [inline README.md:30]:
```suggestion
The `cft-seed` project stores Terraform state and has the Service Account able to create / modify infrastructure.
```

**Pull request merged**